### ISSUE TITLE
Another Cloning Glitch

### ISSUE THREAD

**Issue by CupOfExplodium** [Eclipse-Station/NEV-Northern-Light#1258]
So, once again I have broken cloning.  According to Jazz, a good portion of my code didn't get transferred when I was cloned.  

The result was that I could no longer climb, examine, get on tables, get onto low walls, or do almost anything that requires a skill/stat.  My stats were also reduced to zero.  I couldn't inject myself with syringes (autoinjectors still worked) and I could not eat food, but I no longer needed food.  I also got a massive speed boost, to the point where I could ferry people at very high speeds.  

The biggest issue is that if I got knocked down, I would become unable to get back up.  I would simply become immobilized and helpless.  I wanted to check if enemies would ignore me now, but I got hit by a wire and couldnt get back up.  However, I do believe that hostile mobs might now ignore me since there were spiders ealier that showed no interest in me whatsoever.

Tldr: Cloning broke again, this time it didn't copy some of my code when I got cloned, made me god of speed
.

**Comment by CupOfExplodium** [comment]
Apparently one fix is to just have an admin pop out your soulcrypt, or whatever Jazz did.  
Also getting respawned fixes it too.
